### ISSUE TITLE
`function-parentheses-*-inside` fixes write outside an unclosed function, and grow the file on every run

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#131]
Found while auditing the fix for #114, by running the fixed rule to a fixed point under the value utilities #115 proposes. The fix of an `always` option writes the whitespace a function keeps in front of its closing parenthesis, and where the value parser has marked that function unclosed, the write lands outside the parenthesis instead. Every run adds one more, and no run reports anything.

### What minimal example or steps are needed to reproduce the bug?

```scss
a { transform: translate(1px, 2px // a /*
); }
```

The block comment is never closed, so `postcss-value-parser` reads it as swallowing the closing parenthesis and marks the function `unclosed`.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-scss",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/function-parentheses-space-inside": "always"
  }
}
```

`function-parentheses-newline-inside` does the same under `always` and under `always-multi-line`.

### What did you expect to happen?

Either the whitespace the option asks for, written where the option means it, or nothing at all. A second run should find nothing left to do.

### What actually happened?

`--fix`, four times in a row, `function-parentheses-space-inside` with `always`:

```
a { transform: translate( 1px, 2px // a /*\n) ; }
a { transform: translate( 1px, 2px // a /*\n)  ; }
a { transform: translate( 1px, 2px // a /*\n)   ; }
a { transform: translate( 1px, 2px // a /*\n)    ; }
```

and `function-parentheses-newline-inside` with `always`, where it is a line break that accumulates, so the file grows a blank line per run:

```
a { transform: translate(\n1px, 2px // a /*\n)\n; }
a { transform: translate(\n1px, 2px // a /*\n)\n\n; }
a { transform: translate(\n1px, 2px // a /*\n)\n\n\n; }
```

No warning survives any of the runs. Nothing bounds this: each run reports the problem, counts it as fixed, and leaves the next run the same problem plus one more character.

The cause is `postcss-value-parser`. It marks the function `unclosed: true`, keeps nothing in `after`, and its stringifier prints `after` **outside** the closing parenthesis for such a node, so `valueNode.after = " "` — [`function-parentheses-space-inside/index.js:199-206`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/function-parentheses-space-inside/index.js#L199-L206), and `fixAfterForAlways` in the sibling — puts the space after the parenthesis rather than in front of it. PostCSS then trims that trailing whitespace out of the value and back into the raws of what follows, which is why it reappears in front of the semicolon and why the value looks untouched to the next run. Neither rule asks whether the node it is about to write to is `unclosed`.

### Reachable when?

Only once #115 lands. Today the fix is written into the copy of the value that `postcss-scss` does not print, so it is discarded before it reaches the file — the warning is discarded with it, and the file is left alone. Under `postcss-less` and under plain CSS the fixture does not parse at all (`CssSyntaxError: Unclosed comment`), so no rule runs.

The guard that #113 and #114 put on these two rules neither causes this nor prevents it: a value whose comment never closes has no line break for the guard to reason about, and the guard correctly says that writing there commits nothing out. What is written simply does not go where the rule means it to.

### Do you have a proposal to fix the bug?

Leave a function the value parser has marked `unclosed` alone, in both rules, the same way `isStandardSyntaxFunction` already turns one away: nothing inside a value whose comment never closes can be corrected meaningfully, and the parenthesis the options are about is not there to be moved.
